core/chains/evm/chain.go

```diff
@@ -14,12 +14,10 @@ import (
 	evmconfig "github.com/smartcontractkit/chainlink/core/chains/evm/config"
 	"github.com/smartcontractkit/chainlink/core/chains/evm/types"
 	"github.com/smartcontractkit/chainlink/core/logger"
-	"github.com/smartcontractkit/chainlink/core/service"
```

Comment:
The split between `services`/`service` was there because `services` will pull in a few more services (only balance monitor, prom reporter, it's a lot better now that the v1 code was removed). They should be moved away from the root crate then.

Reply:
This I did not understand. Can you please clarify a little bit more what should I do? Thanks!

Reply:
Do you mean `balance_monitor.*` and `prom_reporter.*` should be moved away from ./services root to respective subfolders?

Reply:
Yeah, nothing should be in the root package other than the service types.

Reply:
Done.